public/src/components/ToastManager.jsx

```diff
@@ -64,28 +68,27 @@ const StyledToast = styled.div`
     }
   }
   @keyframes cancelled {
-  0% {
-    transform: translateX(0px);
-  }
-  90% {
-    transform: translateX(0px);
-  }
-  100% {
-    transform: translateX(300px);
-  }
-}
-  @keyframes immediatecancel {
-  0% {
-    transform: translateX(0px);
+    0% {
+      transform: translateX(0px);
+    }
+    90% {
+      transform: translateX(0px);
+    }
+    100% {
+      transform: translateX(300px);
+    }
   }
-  100% {
-    transform: translateX(300px);
+    @keyframes immediatecancel {
```

Comment:
https://styled-components.com/docs/api#keyframes

I've never used this before, but so you know it's there

Reply:
Ah zank you. that's quite useful.